README.md

```diff
@@ -80,22 +80,21 @@ If you have no need for the UI assets git-lfs is optional.
 
 
 # Component list
-- [`ckcore`](ckcore/) the platform maintaining the [MultiDiGraph](https://en.wikipedia.org/wiki/Multigraph#Directed_multigraph_(edges_with_own_identity)).
-- [`cksh`](cksh/) the Cloudkeeper shell to interact with the core.
-- [`ckui`](ckui/) a UI prototype that can load ckcore exported data but has no backend connection yet.
-- [`ckworker`](ckworker/) provides workers that load [plugins](plugins/) to perform collect and cleanup operations.
-- [`ckmetrics`](ckmetrics/) is a [Prometheus](https://prometheus.io/) [exporter](https://prometheus.io/docs/instrumenting/exporters/).
+- [`resotocore`](resotocore/) the platform maintaining the [MultiDiGraph](https://en.wikipedia.org/wiki/Multigraph#Directed_multigraph_(edges_with_own_identity)).
+- [`resotosh`](resotosh/) the Resoto shell to interact with the core.
+- [`ui`](ui/) a UI prototype that can load resotocore exported data but has no backend connection yet.
+- [`resotoworker`](resotoworker/) provides workers that load [plugins](plugins/) to perform collect and cleanup operations.
+- [`resotometrics`](resotometrics/) is a [Prometheus](https://prometheus.io/) [exporter](https://prometheus.io/docs/instrumenting/exporters/).
```

Comment:
It's really odd that only `ui` and `plugins` aren't prefixed with `resoto`.  I think we should be consistent with the directory names and remove `resoto` (even if the components are still named `resotocore`, `resotosh`, and `resotometrics`)? 🤔

Reply:
The python package that is imported is named with the `resoto` prefix.
Example: `from resotolib.utils import get_local_ip_addresses`
Albeit it looks a bit clumsy I think it is good with prefix.